### ISSUE TITLE
Pynput doesnt release ctrl after hotkey is activated

### ISSUE THREAD

**Issue by not-nef** [not-nef/txt2#9]
Which results in the function getting executed everytime the key gets pressed without ctrl after the hotkey has been activated once

**Comment by not-nef** [comment]
I have no idea how to fix this

**Comment by not-nef** [comment]
So hotkeys will probably become an experimental feature once the settings window is implemented

**Comment by rdbende** [comment]
Just wondering, why do you use `pynput` instead of Tkinter bindings?

**Comment by not-nef** [comment]
@rdbende do they support multiple keys? And if yes pls tell me how i use them

**Comment by rdbende** [comment]
> Do they support multiple keys?

Yes! Events and binds are absolutely one of the most powerful features of Tkinter.

For a multi-key binding you can simply use a hypen `-` as a separator between the keys.
`Ctrl+o` for example: `.bind("<Control-o>", handler)`

But the code above wouldn't really work, as you have to bind every single widget in the app to make it global. Instead you can use the `bind_all` method, which will bind the entire application.
`.bind_all("<Control-o>", handler)`

However, there's still a more elegant approach. You can use the `event_add` method to associate a global keyboard event with a global virtual event that you can bind later.

```python
root.event_add("<<Open>>", "<Control-o>")
root.bind("<<Open>>", handler)
```

**Comment by not-nef** [comment]
Thank you sooooo much!! ❤️

**Comment by Moosems** [comment]
Yeah. Tkinter bindings are super easy to use and make life much easier
